src/config.rs

```diff
@@ -111,12 +112,6 @@ pub fn init_config() -> Result<Config> {
                 .value_name("NODE_COMM_RETRIES")
                 .help("The number of times to retry a node RPC call on any error, set to smaller than 0 for infinite")
                 .takes_value(true))
-        .arg(
```

Comment:
Whether better-call.dev should be used should be optional. This --bcd-enable flag removal should be reverted.

Reply:
Sorry, that commit wasn't meant to be in this PR!